File: 3053/3053.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   3053.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/10/05 15:59:08 by jihoolee          #+#    #+#             */
/*   Updated: 2021/10/05 16:08:05 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>

#define PI	3.14159265358979323846

using namespace std;

int	main(void)
{
	int		r;
	double	euclid_area, taxi_area;

	cin >> r;
	euclid_area = double(r * r) * PI;
	taxi_area = double(r * r) * 2;
	cout << fixed;
	cout.precision(6);
	cout << euclid_area << endl << taxi_area << endl;
	return (0);
}

File: 3053/3053.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    3053.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/10/05 16:08:40 by jihoolee          #+#    #+#              #
#    Updated: 2021/10/05 16:23:46 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

from math import pi

r = int(input())
euclid_area = r ** 2 * pi
taxi_area = r ** 2 * 2
print(euclid_area)
print(taxi_area)
